test/helpers/apiHelper.ts

```diff
@@ -0,0 +1,69 @@
+import axios, { AxiosInstance } from 'axios';
+
+class ApiService {
+    private adminAccessToken: string | null = null;
+    private readonly axiosInstance: AxiosInstance;
+    private readonly apiUrl: string = `${process.env.BASE_URL}api`;
```

Comment:
`${process.env.BASE_URL}api` -> `${process.env.BASE_URL}/api`

Reply:
@AndreyCar, it doesn't work like that, the case of jurl ends like this: ua/